packages/app/src/containers/SignupPage/index.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import {
+    ImageBackground,
+    Text,
+    View,
+    StyleSheet
+} from 'react-native';
+import { Button, Input } from "react-native-elements";
+
+interface SignupPageState {
+    email: string;
+    password: string;
+}
+
+interface OwnProps {
+    navigation: any
+}
+
+type Props = OwnProps;
+
+class SignupPage extends React.Component<Props, SignupPageState> {
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+            email: '',
+            password: '',
+        };
+    }
+
+
+    public componentDidMount() {
```

Comment:
is this being used?

Reply:
@petermsouzajr We do not need it now but we may need this lifecycle method in the future so I just defined this method. I will remove this if you want.